### PR TITLE
adding sorting algo(Bubblesort) :zap:

### PR DESCRIPTION
**Pull request by enkaypeter**
## PR Type
- [x] Snippets, Tests & Tags (new snippets, updated snippets, re-tagging of snippets, added/updated tests)
- [ ] Scripts & Website & Meta (anything related to files in the [scripts folder](https://github.com/30-seconds/30-seconds-of-code/tree/master/scripts), how the repository's automated procedures work and the website)
- [ ] Glossary & Secondary Features (anything related to the glossary, such as new or updated terms or other secondary features)
- [ ] General, Typos, Misc. & Meta (everything else, typos, general stuff and meta files in the repository - e.g. the issue template)

## Guidelines
- [x] I have read the guidelines in the [CONTRIBUTING](https://github.com/30-seconds/30-seconds-of-code/blob/master/CONTRIBUTING.md) document.

### CODE REVIEW

**Comment by skatcat31** [comment]
I see a package lock(which I'll have to confirm what our deployment rules for are), and a testlog(which should not be committed with your work as it's more for build purposes).

I'm also noticing you haven't actually submitted the snippet just yet. It's sometimes helpful to see it in progress while it's developed so you can go back over the code, but it's also sometimes helpful to just upload the MVP in a squash/recommit eventually, so it's really no problem, but it's odd that there would be submission before snippet.

Can you please remove the test log from your submission?

**Comment by enkaypeter** [comment]
Yeah sure. i'll remove the testlogs. And the snippets? my bad :man_facepalming: i didn't add it. I'll do all of those in the next commit. Thanks

**Comment by enkaypeter** [comment]
Done :white_check_mark:

**Comment by skatcat31** [comment]
you'd actually have to remove those changes from history. This can be done a number of ways, but the easiest would be to revert those two changes so there is no change in your branch.

**Comment by enkaypeter** [comment]
That should do yeah?

File: package-lock.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
```
Comment (Chalarangelo) [inline package-lock.json]: Please unstage all changes to `package.json`.

File: snippets/bubbleSort.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+###bubbleSort
```
Comment (Chalarangelo) [inline snippets/bubbleSort.md]: `bubbleSort` is more of an algorithmic implementation than an everyday-use snippet, as there is already a native function built into JS. This snippet can be merged, but will have to be moved to the archive.

File: tag_database [hunk 3]
```diff
@@ -17,6 +17,7 @@ bindKey:function,object,intermediate
 binomialCoefficient:math,intermediate
 bottomVisible:browser,intermediate
 btoa:node,string,utility,beginner
+bubbleSort:uncategorized
```
Comment (Chalarangelo) [inline tag_database]: As the snippet will be moved to archive, tags are probably not needed, so the scripts will automatically handle these. Nothing to worry about in this file.

File: snippets/bubbleSort.md (hunk 2 above)
Comment (enkaypeter) [reply snippets/bubbleSort.md]: Alright

File: tag_database (hunk 3 above)
Comment (enkaypeter) [reply tag_database]: Ok.

**Comment by enkaypeter** [comment]
> @enkaypeter https://stackoverflow.com/questions/1817766/how-to-revert-to-origins-master-branchs-version-of-file
> 
> WARNING: be careful and read closely. This does have an example about resetting wholly to master.

Hey, did what was stated in the stackoverflow thread.

**Comment by enkaypeter** [comment]
May be i'm not getting what need to be done. The `testlog` file no longer on my branch or the `master` branch. It has been checked out. Could you help point me in the right direction?

**Comment by skatcat31** [comment]
Have you committed and pushed the checked out change so the changes are no longer in the branch?

**Comment by enkaypeter** [comment]
Yeah i have

**Comment by skatcat31** [comment]
It looks like you have not committed and pushed checking out the testlog file yet. Doing that should solve the problem

**Comment by skatcat31** [comment]
Wait, for clarification what do you mean is no longer on the `master` branch?

**Comment by enkaypeter** [comment]
Yeah, on the Master branch and on My branch

**Comment by skatcat31** [comment]
I'm going to close the PR due to the time spent attempting to fix conflicts. We'd welcome the changes in a new branch that does not attempt to commit the built files, and just brings in the changes. Thank you for the understanding, but at this point it's hard to remotely debug, and would be faster to copy the changes to a new branch and to abandon the confusing history to the ether.

**Comment by skatcat31** [comment]
basically 30 seconds of copy and pasting files, versus possibly trying to fix history for hours. We'd suggest starting with a clean copy of the repository as well.

With that said don't let this deter you. This is something I've done a lot with developers(both new and old hats) who are learning Git or are even very experienced. Often times it's better to start over and copy the files then it is to try and fix the history using the tools and getting confused, leading to a much more dangerous situation.

**Pull request closed without merging**